src/app/mypage/page.tsx

```diff
@@ -14,7 +14,7 @@ export default function MyPage() {
           <ProjectRegisterButton text="새 프로젝트 등록하기" className="h-[45px] w-[210px]" />
         </div>
         <h2 className="text-gray-900 body6">프로필</h2>
-        <UserProfile fromMyProfile />
+        <UserProfile fromMyProfile iconIndex={0} />
```

Comment:
```suggestion
        <UserProfile fromMyProfile />
```
UserProfile은 0 고정인데 부모로부터 prop을 받지 않아도 될거같아요
UserProfile 내에서 0으로 아이콘 컴포넌트에 prop 보내주는걸로 충분할 것 같습니다

Reply:
✅ [a77299b](https://github.com/PRism-Works/PRism-FE/pull/96/commits/a77299b535b60ca3068a556b76e3d0289173413c)